src/app/components/Btn.js

```diff
@@ -0,0 +1,23 @@
+export const Btn = (
```

Comment:
W programowaniu ważniejsza jest jasność przekazu niż nawet długość. O ile nawet w tym przypadku nic nie zyskujmy pisząc Btn zamiast Button (jedynie niepewność czy to na pewno Button). Więc lepiej to nazwać Button.js.

Reply:
W powyższym przypadku zgodzę się (bo to nazwa pliku/komponentu).
Choć w nazwach zwykłych zmiennych lub funkcji będę się wspierał ;)

Reply:
Czym będziesz się wspierał? Moją radą :P ? 
Jak zawsze "to zależy". Ale długa jeszcze nigdy nikomu nie zaszkodziła :)

Reply:
Teraz sobie myślę, że ciężko się wspierać.
Ja nie miałem możliwości pracować z dużą liczbą programistów nad wspólnym kodem, więc powinienem się domyśleć, że mogą powstać "niezłe kwiatki", skoro o tym piszesz :)